app/controllers/microposts_controller.rb

```diff
@@ -0,0 +1,27 @@
+class MicropostsController < ApplicationController
+  # GET /microposts
+  def index
+    @micropost = Micropost.new
+    @microposts = Micropost.order(id: :desc).page params[:page]
```

Comment:
```suggestion
    @microposts = Micropost.order(id: :desc).page(params[:page])
```

【好み】
メソッドチェーンには括弧ほしい。

Reply:
最後のメソッドの引数だけ括弧なしなのは一貫性がないように感じますね。
メソッドチェーンの記法を使うときは、括弧は省略しないようにします。